### PR TITLE
add list on javascript mentor

### PR DESCRIPTION
**Pull request by aibrahim3546**
Hey, there new mentor here. I have added my name on the list

### CODE REVIEW

File: mentors/javascript/a.json [hunk 1]
```diff
@@ -38,5 +38,13 @@
...
     "avatar_url": "https://avatars1.githubusercontent.com/u/5274697?s=460&v=4",
     "bio": "I'm one part professional developer and two parts programming enthusiast, shaken with a liberal dash of pragmatism."
+  },
+  {
+    "github_username": "aibrahim3546",
+    "name": "Ahmad Ibrahim",
+    "link_text": "https://aibrahim3546.github.io/",
+    "link_url": "https://aibrahim3546.github.io/",
+    "avatar_url": null,
+    "bio": "I Javascript developer, ReactJs, NodeJs, React Native."
```
Comment (rpottsoh) [inline mentors/javascript/a.json]:
```suggestion
    "bio": "I am a Javascript developer, ReactJs, NodeJs, React Native."
```

You may want to clean up your "bio" sentence some more, so it makes more sense.  I am not sure if you are just a Javascript developer and also a developer that works with ReactJS, NodeJS, and React Native?  Let me know if you would like some help with the wording.

Comment (aibrahim3546) [reply mentors/javascript/a.json]: I have made some adjustment. Have a look.
Thank you for the suggestion. :)

File: mentors/javascript/a.json [hunk 2]
```diff
@@ -45,6 +45,6 @@
     "link_text": "https://aibrahim3546.github.io/",
     "link_url": "https://aibrahim3546.github.io/",
     "avatar_url": null,
-    "bio": "I Javascript developer, ReactJs, NodeJs, React Native."
+    "bio": "I am a fan of javascript. I have been working with ReactJs for a year and yeah I am really good at ReactJs. Also Good at NodeJs and React Native"
```
Comment (rpottsoh) [inline mentors/javascript/a.json]: Nice!  Welcome to Exercism as a javascript mentor!

Comment (aibrahim3546) [reply mentors/javascript/a.json]: Thanks mate

**Pull request merged**